containers/ddev-dbserver/Dockerfile

```
@@ -33,32 +40,56 @@ RUN set -x; if ( command -v mariadbd && ! command -v mysqld ); then \
 # look there when doing apt-get update. And we don't use new packages from there.
 # And we're going to install our own percona.list if needed, so get that if needed
 # and remove it here
+USER root
 RUN rm -f /etc/apt/sources.list.d/mariadb.list /etc/apt/sources.list.d/percona.list
+RUN mkdir -p /var/lib/apt/lists/partial
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y -o Dpkg::Options::="--force-confold" bzip2 curl gnupg2 less lsb-release pv tzdata vim-tiny wget
```

We can add `vim` alternative, as in #6839

```suggestion
RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y -o Dpkg::Options::="--force-confold" bzip2 curl gnupg2 less lsb-release pv tzdata vim-tiny wget
RUN update-alternatives --install /usr/bin/vim vim /usr/bin/vim.tiny 10
```